README.md

```diff
@@ -272,6 +283,48 @@ func (Executor) Run(l *log.Entry, aliases venom.Aliases, step venom.TestStep) (v
 Feel free to open a Pull Request with your executors.
 
 
+## TestCase Context
+
+TestCase Context allows you to inject datas in all Steps
```

Comment:
Add: "define a context is optional, but can be usefull to keep data between teststeps on a testcase."
?